CONTRIBUTING.rst

```diff
@@ -33,34 +30,55 @@ You'll need to fork this repository and get the
 
 * Look for issues with the `good first issue
   <https://github.com/catalyst-cooperative/pudl/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue>`__
-  tag. These are issues that don't require a ton of PUDL-specific context, and
-  are relatively tightly scoped to boot.
+  tag in our `Community Kanban Board
+  <https://github.com/orgs/catalyst-cooperative/projects/9/views/19>`__. These
+  are issues that don't require a ton of PUDL-specific context, and are
+  relatively tightly scoped.
 
 * Comment on the issue and tag ``@com-dev`` (our Community Development Team) to
   let us know you're working on it. Feel free to ask any questions you might
   have!
 
 * Once you have an idea of how you want to tackle this issue, write out your
-  plan so we can guide you around obstacles in your way.
+  plan so we can guide you around obstacles in your way! Post a comment outlining:
+  * what steps have you broken this down into?
+  * what is the output of each step?
+  * how will one know that each step is working?
```

Comment:
Should we ask people to wait for response / ping com-dev here before continuing / starting on the project? This looks good though!